e2e/capacity/transactions.test.ts

```diff
@@ -245,11 +245,15 @@ describe('Capacity Transactions', function () {
         before(async function () {
           capacityKeys = createKeys('CapacityKeys');
           capacityProvider = await createMsaAndProvider(fundingSource, capacityKeys, 'CapacityProvider', FUNDS_AMOUNT);
+          // Create a MSA for the delegator
```

Comment:
Do it once instead of over and over.

Reply:
I prefer keeping them separate as it makes debugging easier. I believe they were written this way at one point. However, this is fine since I don’t see these extrinsics to changing much.

Reply:
Yeah. It really bogs down the runs on Testnet.